sunpy/timeseries/sources/goes.py

```diff
@@ -274,6 +277,9 @@ def _parse_netcdf(filepath):
             elif "xrsa_flux" in d.variables:
                 xrsa = np.array(d["xrsa_flux"])
                 xrsb = np.array(d["xrsb_flux"])
+                xrsa_quality = np.array(d['xrsa_flags'])
+                xrsb_quality = np.array(d['xrsb_flags'])
```

Comment:
Are there other columns that would be useful to have extracted too?

Reply:
other variables in netcdf - not sure if any of those should also be exposed:

```
xrsa1_flux: ('time',) float32
xrsa2_flux: ('time',) float32
xrsb1_flux: ('time',) float32
xrsb2_flux: ('time',) float32
xrsa_primary_chan: ('time',) uint8
xrsb_primary_chan: ('time',) uint8
xrsa1_flags: ('time',) uint16
xrsa2_flags: ('time',) uint16
xrsb1_flags: ('time',) uint16
xrsb2_flags: ('time',) uint16
corrected_current_xrsa2: ('time', 'quad_diode') float32
corrected_current_xrsb2: ('time', 'quad_diode') float32
au_factor: ('time',) float32
roll_angle: ('time',) float32
yaw_flip_flag: ('time',) uint8
```

Reply:
I think we can leave these, unless someone else requests one of them with a reason.